### DISCUSSION TITLE
英文封面只显示作者名和导师名，学院、专业等信息没有显示，这是为什么？怎么解决？

### DISCUSSION THREAD

**Discussion by WEIYujie1996** [sjtug/SJTUThesis#1056] [提问]
英文封面只显示作者名和导师名，学院、专业等信息没有显示，这是为什么？怎么解决？
选择的模板是：\documentclass[type=doctor,lang=en,math-style=TeX]{sjtuthesis}

**Comment by LogCreative** [comment]
学院信息在最下面一栏，需要通过设置指定：
```latex
\sjtusetup{info/en/department={Department of XXX}}
```

根据学校的官方模板，专业信息不显示在英文封面上。
